suites/portal/indexingPageTest.js

```diff
@@ -61,7 +61,7 @@ Scenario('Navigate to the indexing page and upload a test manifest @indexing', a
   I.waitForElement({ css: '.indexing-page' }, 10);
   I.click('.index-flow-form'); // after clicking open window file upload dialog
   await I.attachFile('input[type=\'file\']', `manifest_${I.cache.UNIQUE_NUM}.tsv`);
-  I.click({ xpath: 'xpath: //button[contains(text(), \'Index Files\')]' });
+  await I.click({ xpath: 'xpath: //button[contains(text(), \'Index Files\')]' });
```

Comment:
I.click does not seem to be asynchronous from the docs ... can you check that once

Reply:
I've noticed that the `checkPod` log entries were showing up before the click event logs, hence, the introduction of the `await` 🤷